table.csv

```diff
@@ -35,6 +35,7 @@ multihashes,,
 sha1,               ,                         0x11
 sha2-256,           ,                         0x12
 sha2-512,           ,                         0x13
+dbl-sha2-256,       ,                         0x56
```

Comment:
Yeah, nevermind what I was proposing then, I'm just afraid that it might get out of control.